nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -41,6 +41,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - Invidious has changed its default database username from `kemal` to `invidious`. Setups involving an externally provisioned database (i.e. `services.invidious.database.createLocally == false`) should adjust their configuration accordingly. The old `kemal` user will not be removed automatically even when the database is provisioned automatically.(https://github.com/NixOS/nixpkgs/pull/265857)
 
+- The legacy and long deprecated systemd target `network-interfaces.target` has been removed.
```

Comment:
```suggestion
- The legacy and long deprecated systemd target `network-interfaces.target` has been removed. Use the `network.target` instead.
```

I think since it was basically just wrapping that, and `network.target` has those exact semantics (network interfaces are now available) this is a sound recommendation.

Reply:
It should just be ``Use `network.target` instead.`` omitting “the” from the sentence, right?